odh/overlays/moc/monitoring/dashboards/thoth-station/thoth-service-metrics.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata:
+  name: thoth-user-api-metrics
```

Comment:
Name doesn't match the file. I think you forgot to change this

Reply:
Ah.. yes... @HumairAK is faster again.. https://github.com/operate-first/apps/pull/191#issuecomment-770904582